slapo/model_schedule/api.py

```diff
@@ -0,0 +1,85 @@
+# Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+# SPDX-License-Identifier: Apache-2.0
+
+from .registry import get_schedule_method
+from ..schedule import create_schedule
+from ..logger import get_logger
+
+
+def apply_schedule(
```

Comment:
This function specifies a schedule template and calls model specify methods, but it seems not flexible at the first glance. However it might be sufficient for now so I'll need to think more on this.